integtest/included.asciidoc

```diff
@@ -0,0 +1 @@
+I am tiny.
```

Comment:
![i am tiny](https://pics.me.me/i-am-a-tiny-potato-and-i-believe-in-you-3535009.png)